### PR TITLE
Final Recipe Group Project 

### PR DESCRIPTION
**Pull request by Ridwanullahi-code**
@Ridwanullahi-code  and @Hisoka37 worked on this project

- [ ] Login page and registration page:
- [ ] Should be built with Devise.
- [ ] Food list (only for a group that has less than 3 members):
- [ ] Should display a list of food added by the logged-in user as in the wireframe (display also quantity of a given food).
- [ ] Should lead to a form that allows users to add new food.
- [ ] Recipes list:
- [ ] Should display a list of recipes created by the logged-in user as in the wireframe.
- [ ] Should lead to recipe details.
- [ ] If the user is the owner of the recipe, should allow the user to delete it.
- [ ] Public recipe list:
- [ ] Should display a list of all public recipes ordered by newest as in the wireframe.
- [ ] Should lead to recipe details.
- [ ] Recipe details:
- [ ] Should display a toggle button that allows for a recipe to be made public or private.
- [ ] If the recipe is public or the user is the owner of the recipe, should display the recipe details as in the wireframe.
- [ ] If the user is the owner of the recipe, should lead to the form that allows the user to add new food.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,161 @@
...
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## :pray: Acknowledgments <a name="acknowledgements"></a>
+I would like to thank... <a href="https://www.microverse.org/?gclid=CjwKCAiArY2fBhB9EiwAWqHK6s-2-x4d57Pghz47XT1BgsYuF81ZprM-k-IwzI0_L96nV0SQ93A8ExoCVnQQAvD_BwE" title="planet icons">Microvers for giving me this opportunity to build this project</a>
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (VICTOR-CHUKWU) [inline README.md:159]: - [ ] your MIT link is broken, kindly fix that

File: app/views/recipes/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    <tr>
+      <th>Food</th>
+      <th>Quantity</th>
+      <th>Value</th>
+      <th>Actions</th>
+    </tr>
+  </thead>
+  <tbody>
+    <% @ingredients.each do |recipe_food| %>
+      <tr>
```
Comment (VICTOR-CHUKWU) [inline app/views/recipes/show.html.erb:41]: - [ ] kindly try to match the wireframe given, I can not see a button to toggle the public recipe, also the button destroy and edit is not necessary as it is not in the wireframe. kindly adjust this view
in your design :point_down: 
Screenshot from 2023-07-07 11-17-03
wireframe :point_down: 
Screenshot from 2023-06-29 12-00-33

File: app/views/shared/_navbar.erb [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+            <%= button_to "Sign Out",
+            destroy_user_session_path,
+            method: :delete,
+            class: "btn btn-small btn-outline" %>
+          <% else %>
+            <%= button_to "Sign In", new_user_session_path%>
+          <% end %>
+      </ul>
+    </div>
+  </div>
```
Comment (VICTOR-CHUKWU) [inline app/views/shared/_navbar.erb:38]: - [ ] good job on the drop downs :clap:

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @Ridwanullahi-code  and @Hisoka37  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed
- projects run correctly
- devise setup correctly


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Hisoka37) [reply README.md:159]: Fixed 👌🏻

File: app/views/recipes/show.html.erb (hunk 2 above)
Comment (Hisoka37) [reply app/views/recipes/show.html.erb:41]: Fixed 👌🏻

**Review by omarsalem7 — APPROVED** [review]
Hi @Ridwanullahi-code 😊, 
# STATUS: APPROVED! 🥇 💯 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**